lib/components/narrative/default/itinerary-summary.js

```diff
@@ -47,6 +47,7 @@ const Block = connect(mapStateToProps)(styled.div`
 
   span {
     display: ${(props) => (props.renderRouteNamesAndColors ? 'block' : 'none')};
+    margin-top: -3px;
```

Comment:
This fixes things, but I don't know what it's accounting for... feels risky

Reply:
`align-self: center` works!!